messaging/src/main/java/org/axonframework/commandhandling/gateway/AxonNonTransientExceptionClassesPredicate.java

```diff
@@ -0,0 +1,20 @@
+package org.axonframework.commandhandling.gateway;
+
+import org.axonframework.common.AxonNonTransientException;
+
+/**
+ * An Axon framework specific predicate for checking non-transiency of a failure comparing it against concrete classes.
+ * <p/>
+ * This implementation uses only {@link AxonNonTransientException} class for comparisons.
+ * <p/>
+ * This predicate is intended to be used for configuring {@link RetryScheduler} implementations where it is used as a
+ * default if the user does not explicitly reconfigure it.
```

Comment:
Sorry for providing an updated piece of JavaDoc here. We've quite recently moved to Grammarly, which has upgraded the new pieces of Javadoc.
Just did the same with this section, which resulted in the following:
```suggestion
 * An Axon-specific {@link Predicate}, used to check the non-transiency of a failure comparing it against concrete classes.
 * <p/>
 * This implementation uses only {@link AxonNonTransientException} class for comparisons.
 * <p/>
 * This {@code Predicate} acts as the default for {@link RetryScheduler} instances.
```